src/components/Chip/Chip.tsx

```diff
@@ -64,16 +64,22 @@ export const Chip = ({
         removable && styles['Chip--removable'],
         className,
       )}
-      role="option"
-      aria-label={title}
       {...restProps}
     >
       <div className={styles['Chip__in']} role="presentation">
-        {hasReactNode(before) && <div className={styles['Chip__before']}>{before}</div>}
-        <Footnote className={styles['Chip__content']} title={title} aria-hidden>
+        {hasReactNode(before) && (
+          <div className={styles['Chip__before']} aria-hidden>
```

Comment:
Кажется что aria-hidden тут не нужен

```suggestion
          <div className={styles['Chip__before']}>
```

Пусть делают на самом before если надо